PaTTAN Room Displays/PaTTAN Room Displays/App.xaml.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace PaTTAN_Room_Displays;
+﻿using System.Threading.Tasks;
+
+namespace PaTTAN_Room_Displays;
```

Comment:
https://stackoverflow.com/questions/45313834/whats-the-name-for-a-mix-of-snake-case-and-camel-case

Reply:
I inherited that, but it's an easy fix?  What do you suggest?  pattanRoomDisplays?

Reply:
Pascal Case: PattanRoomDisplays

Reply:
we might be stuck with the name space PaTTAN_Room_Display the IDE won't change it. Normally I'd assume I could just change it on each page, but MS has a way of proving me wrong.